src/metabase/db/data_migrations.clj

```diff
@@ -159,51 +153,7 @@
 (defmigration ^{:author "senior", :added "0.30.0"} clear-ldap-user-local-passwords
   (db/transaction
     (doseq [user (db/select [User :id :password_salt] :ldap_auth [:= true])]
-      (db/update! User (u/the-id user) :password (creds/hash-bcrypt (str (:password_salt user) (UUID/randomUUID)))))))
-
-
-;; In 0.30 dashboards and pulses will be saved in collections rather than on separate list pages. Additionally, there
-;; will no longer be any notion of saved questions existing outside of a collection (i.e. in the weird "Everything
-;; Else" area where they can currently be saved).
-;;
-;; Consequently we'll need to move existing dashboards, pulses, and questions-not-in-a-collection to a new location
-;; when users upgrade their instance to 0.30 from a previous version.
-;;
-;; The user feedback we've received points to a UX that would do the following:
-;;
-;; 1. Set permissions to the Root Collection to readwrite perms access for *all* Groups.
-;;
-;; 2. Create three new collections within the root collection: "Migrated dashboards," "Migrated pulses," and "Migrated
-;;    questions."
-;;
-;; 3. The permissions settings for these new collections should be set to No Access for all user groups except
-;;    Administrators.
-;;
-;; 4. Existing Dashboards, Pulses, and Questions from the "Everything Else" area should now be present within these
-;;    new collections.
-;;
-(defmigration ^{:author "camsaul", :added "0.30.0"} add-migrated-collections
-  (let [non-admin-group-ids (db/select-ids PermissionsGroup :id [:not= (u/the-id (perm-group/admin))])]
-    ;; 1. Grant Root Collection readwrite perms to all Groups. Except for admin since they already have root (`/`)
-    ;; perms, and we don't want to put extra entries in there that confuse things
-    (doseq [group-id non-admin-group-ids]
-      (perms/grant-collection-readwrite-permissions! group-id collection/root-collection))
-    ;; 2. Create the new collections.
-    (doseq [[model new-collection-name] {Dashboard (trs "Migrated Dashboards")
-                                         Pulse     (trs "Migrated Pulses")
-                                         Card      (trs "Migrated Questions")}
-            :when                       (db/exists? model :collection_id nil)
-            :let                        [new-collection (db/insert! Collection
-                                                          :name  new-collection-name
-                                                          :color "#509ee3")]] ; MB brand color
-      ;; 3. make sure the non-admin groups don't have any perms for this Collection.
-      (doseq [group-id non-admin-group-ids]
-        (perms/revoke-collection-permissions! group-id new-collection))
```

Comment:
It's not needed. The default groups could have got perms for these collections when we created them because of Toucan `post-insert` behavior but since we're doing things in Liquibase now all we need to do is **not** create entries in the `permissions` table for the newly created Collection.